packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -104,6 +109,7 @@ export function createApp() {
       'export/dashboardVisualisation',
       handleWith(ExportDashboardVisualisationRoute),
     )
+    .post<ExportDataTableRequest>('export/dataTable', handleWith(ExportDataTableRoute))
```

Comment:
What's the context where we're exporting without an id? Feels like we should always expect one

Reply:
Oh good catch. Some of our exports allow a `POST` with a list of ids in the body, but I decided against including that here. I must've missed deleting this line